src/schemas/ProgramDonorSummary/resolvers/summaryStats.ts

```diff
@@ -140,6 +148,7 @@ const programDonorSummaryStatsResolver: (
       donorsInvalidWithCurrentDictionary: FilterAggregationResult;
       allFilesCount: NumericAggregationResult;
       filesToQcCount: NumericAggregationResult;
+      lastUpdate?: DateAggregationResult;
```

Comment:
Low cost add, might use this in a UI where the DCC can review if they want to trigger the donor re-index.